modules/apps/batch-engine/batch-engine-service/bnd.bnd

```diff
@@ -4,8 +4,18 @@ Bundle-Version: 4.0.62
 Import-Package:\
 	!com.graphbuilder.*,\
 	\
+	!de.rototor.pdfbox.graphics2d.*,\
```

Comment:
@javalosjr96 I do not think this is something I can get by "Auto SF" according to your commit message. Could you please separate the changes you made manually?

Reply:
Ok.

Reply:
@tinatian Commit was split.